weather-app/src/app/services/day-time.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DayTimeService {
+
+  constructor() { }
```

Comment:
you can remove constructor and ngOnInit if you don't need them here

Reply:
Fixed.